lib/package.gi

```diff
@@ -1192,6 +1192,9 @@ InstallGlobalFunction( ReadPackage, function( arg )
     if   Length( arg ) = 1 then
       # Guess the package name.
       pos:= Position( arg[1], '/' );
+      if pos = fail then
+        Error(Concatenation(arg[1], " is not a valid filename. Use 'LoadPackage' to load packages"));
+      fi;
```

Comment:
I'll change the message. I have seem other people do this, including myself. I think the problem is that `Read` is the standard function for reading a file, so `ReadPackage` seems natural for reading a package (as opposed to a single file from a package). I do agree the message could be more helpful for actual users of `ReadPackage`